tests/test_preprocessing.py

```diff
@@ -114,6 +114,46 @@ def test_pipeline_stopwords(self):
         pipeline = [preprocessing.lowercase, preprocessing.remove_stopwords]
         self.assertEqual(preprocessing.clean(s, pipeline=pipeline), s_true)
 
+    """
+    Test clean
+    """
+
+    def _get_default_clean_pipeline(self):
```

Comment:
Not relevant for this PR, right?

Reply:
we should still keep a clean unit test, but I can remove the get default clean pipeline from here.